README.md

```diff
@@ -1,18 +1,20 @@
 # REDI Sprint 2020 Frontend Development 2, Final Project
 
-Application Name: {app-name}
+Application Name: {Weather App}
 
-Student Name: {student-name}
+Student Name: {Vidhya Krishnan}
 
 ## {app-name}
 
-{short explanation of your application}
+{Weather app which accepts user information in the form of placenames and displays the weather of the respective place}
 
 ## Screenshots
 
-![screen shot](docs/screen_cap.png)
+![screen shot](C:\Final Project\js_sprint_2020_final_project_vidhya-eng\outline.PNG)
```

Comment:
This won't work. You need set path related to the project like it was before `docs/screen_cap.png`